stdlib/public/runtime/Errors.cpp

```diff
@@ -22,11 +22,80 @@
 #include <pthread.h>
 #include <stdarg.h>
 #include "swift/Runtime/Debug.h"
+#include "swift/Basic/Demangle.h"
+#include "swift/Config.h"
+#include <regex>
+#include <cxxabi.h>
+#include <iostream>
+#include <execinfo.h>
 
 #ifdef __APPLE__
 #include <asl.h>
 #endif
 
+static std::string
+cppDemangle(std::string line, bool *success)
+{
+  std::regex cppRegex("(.*?)(_Z[_a-zA-Z0-9$]+)(.*)");
```

Comment:
Symbol names on Darwin platforms have an extra leading `_` (i.e. __Z…). You should handle that here. I don't know if there is an existing #if to check.